File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for command in commands:
        i = command[0]
        j = command[1]
        k = command[2]
        
        newArray = array[i-1:j]
        newArray.sort()
        answer.append(newArray[k-1])
    
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.kt
class Solution {
    fun solution(array: IntArray, commands: Array<IntArray>): IntArray {
        var answer = mutableListOf<Int>()
        for((i,j,k) in commands){
            val cut = array.copyOfRange(i-1, j)
            cut.sort()
            answer.add(cut[k-1])
        }
        return answer.toIntArray()
    }
}